autonav_ws/src/autonav_msgs/CMakeLists.txt

```diff
@@ -12,15 +12,19 @@ find_package(builtin_interfaces REQUIRED)
 
 # generate messages
 set(msg_files
+    "msg/AngularPIDStatistics.msg"
     "msg/AudibleFeedback.msg"
     "msg/Conbus.msg"
     "msg/ControllerInput.msg"
     "msg/DeviceState.msg"
     "msg/GPSFeedback.msg"
     "msg/IMUData.msg"
+    "msg/LinearPIDStatistics.msg"
     "msg/Log.msg"
     "msg/MotorFeedback.msg"
     "msg/MotorInput.msg"
+    "msg/MotorStatisticsBackMotors.msg"
+    "msg/MotorStatisticsFrontMotors.msg"
```

Comment:
why are these separate messages? aren't both back and front motor messages the same? they should be different callbacks yes but same message type